broker/broker.go

```diff
@@ -193,7 +197,7 @@ func (b *Broker) Provision(ctx context.Context, instanceID string, details broke
 	}, nil
 }
 
-// Update modifies an existing service instance
+// Update modifies an existing service instance. It can call up to two aws elasticache commands synchronously dependent upon the parameters provided
```

Comment:
:confused: I'm not sure I get this comment. What happens if I try specifying _three_ parameters? Does it then have to do it asynchronously? Is someone going to remember to come along and change the number to three if they add support for another parameter? If it's done synchronously, why is it returning `IsAsync: true,` in the successful case?

Reply:
It is not proper async operation. It merely tells the broker to return a 202 if the request has been accepted and is being processed. Don't you think it's a liberty describing that as asynchronous operation?